lib/active_merchant/billing/affirm.rb

```diff
@@ -0,0 +1,49 @@
+require 'affirm'
+
+module ActiveMerchant #:nodoc:
```

Comment:
We use both here, the `Spree::Gateway` wraps this one so I can test the AM gateway in isolation of Spree.